pkg/ebpf/tracee.go

```diff
@@ -783,6 +784,29 @@ func (t *Tracee) populateBPFMaps() error {
 				return err
 			}
 		}
+		if e == PrintSyscallTableEventID {
```

Comment:
This loop is for tail-calls/params map initialization (which is generic for all events).
Let's handle this outside of this loop (can be done before)